WebDriverAgentLib/Categories/XCUIElement+FBScrolling.m

```diff
@@ -279,4 +281,23 @@ - (BOOL)fb_scrollAncestorScrollViewByVectorWithinScrollViewFrame:(CGVector)vecto
   return didSucceed;
 }
 
+- (XCElementSnapshot *)fb_findVisibleScrollableView
+{
+  NSArray *possibleParents = @[
+                               @(XCUIElementTypeScrollView),
+                               @(XCUIElementTypeCollectionView),
+                               @(XCUIElementTypeTable),
+                              ];
+    
+  XCElementSnapshot *scrollView = [self fb_parentMatchingOneOfTypes:possibleParents];
+    
+  if (scrollView == nil) {
+    return nil;
+  } else if (![scrollView fb_isVisible]) {
```

Comment:
Please use fast returns. `else` is not needed here as you `return` inside `if` anyway. It will simplify the code